pkg/crypto/internal/didopts/didopts.go

```diff
@@ -0,0 +1,12 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package didopts
+
+// CreateDIDOpts holds the options for creating DID
+type CreateDIDOpts struct {
```

Comment:
Question: this used to be private to the wallet package, and with the package being split up I needed to pull this out into an internal package - question is, does this need to be internal to `pkg/crypto`? If I move this to, for example, `pkg/internal/didutil` then what I've moved into `pkg/crypto/didcreator` can be merged into `pkg/common/did`.

Reply:
I don't see why this needs to be in the wallet.